### PR TITLE
completed javascript forms

### PR DESCRIPTION
**Pull request by KennethDelaCruz**
image

### CODE REVIEW

**Comment by KennethDelaCruz** [comment]
forgot to add the first photo!
image

File: javascript-forms/form-events.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+function handleInput(event) {
+  console.log('this is logging the name and value', event.target.name, event.target.value);
+}
+
+var userName = document.querySelector('#user-name');
+var userEmail = document.querySelector('#user-email');
+var userMessage = document.querySelector('#user-message');
+
+console.log();
```
Comment (codingcodymiller) [inline javascript-forms/form-events.js]: ❓ ❓ ❓ ❓ ❓

File: javascript-forms/form-values.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+var formRoute = document.querySelector('form');
```
Comment (codingcodymiller) [inline javascript-forms/form-values.js]: This can be removed

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far! A couple notes:

File: javascript-forms/form-values.js [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
-    name: formRoute.name.value,
-    email: formRoute.email.value,
-    message: formRoute.message.value
+    name: name1,
+    email: email1,
+    message: message1
   };
   console.log('form information', formInfo);
-  formRoute.reset();
+  document.querySelector('form').reset();
```
Comment (codingcodymiller) [inline javascript-forms/form-values.js]: You can just use `contactForm` instead of querying the DOM again

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
One more change!

**Review by codingcodymiller — APPROVED** [review]
👏 👏 👏

**Pull request merged**